incuna-sass/mixins/_font-face.sass

```diff
@@ -0,0 +1,59 @@
+// New font face mixin with more flexibility and more comprehensive browser support
+// Requires that you set the font file path for the project locally
+// Like this:
+//      @function font-url($path)
+//          @return url('../path/to/fonts/' + $path)
+@mixin font-face($style-name, $file, $family, $category: '', $woff2: false, $IE-8: false)
+    $filepath: $family + '/' + $file
+    @font-face
+        font-family: '#{$style-name}'
+
+        // Default src without IE-8 and woff2 support
+        src: font-url($filepath + ".woff") format('woff'), font-url($filepath + ".otf") format('opentype')
+
+        // Support for woff2 files is optional 
+        // (this format still has quite limited support)
+        @if $woff2
+            src: font-url($filepath + '.woff2') format('woff2'), font-url($filepath + ".woff") format('woff'), font-url($filepath + ".otf") format('opentype')
```

Comment:
Does this actually output two `src:` properties in the CSS, or does Sass overwrite the first one with the ones in these if blocks?

I think IE8 has some trouble recognising multiple src properties so we probably want to make sure there is only ever one line output